src/coreclr/src/debug/daccess/enummem.cpp

```diff
@@ -1450,7 +1465,8 @@ HRESULT ClrDataAccess::DumpStowedExceptionObject(CLRDataEnumMemoryFlags flags, C
 
     CATCH_ALL_EXCEPT_RETHROW_COR_E_OPERATIONCANCELLED
     (
-        ReportMem(vTableAddress + sizeof(PBYTE)* TEAR_OFF_SLOT, sizeof(TADDR));
+        ReportMem(vTableAddress, sizeof(TADDR)); // Report the QI slot on the vtable for ComWrappers
```

Comment:
We can't add an `#ifdef` in the parentheses because this is a macro invocation. I can refactor out just these two `ReportMem` calls so we can correctly ifdef the calls if you'd prefer.

Reply:
ah I wasn't thinking about that. Don't worry about it, the comment is already a pretty good marker and if we captured one extra pointer in the future in a config with this feature disabled its not a big deal : )